endpoints/openrtb2/auction_test.go

```diff
@@ -227,16 +229,74 @@ func TestBlacklistRequests(t *testing.T) {
 	tests.assert(t)
 }
 
+// TestRejectAccountRequired asserts we return a 400 code on a request that comes with no user id nor app id
+// if the `AccountRequired` field in the `config.Configuration` structure is set to true
+func TestRejectAccountRequired(t *testing.T) {
+	tests := map[string]*getResponseFromDirectory{
+		"8.1) Not required and not provided. Since not provided, not blacklisted": {
```

Comment:
What does these numbers `8.1`, `8.2`, etc mean? Also, why are these test cases part of a map? Doesn't look like we're using the key anywhere. Would recommend just making it a slice and you can comment out the description of each test case so its easier to understand for anyone else. Ideally with table tests we should have a `description` field that is printed in case of failed assertions.

Reply:
Mansi I converted the map into a slice and just left the description commented out since the errors themselves come with an error message. Let me knwo what you think